packages/hop-node/src/vault/Vault.ts

```diff
@@ -9,6 +10,13 @@ export abstract class Vault {
     if (useYearnVault) {
       return new YearnVault(chain, token, signer)
     }
+
+    // TODO: determine what tokens and chain to use aave vault for
+    const aaveVaultTokens = new Set(['USDC'])
+    const useAaveVault = chain === Chain.Arbitrum && aaveVaultTokens.has(token)
+    if (useAaveVault) {
+      return new AaveVault(chain, token, chain)
```

Comment:
The last param looks like a duplicate and should be `signer`. It will be the GasBoost signer, correct?

Reply:
nice catch, fixed